virtualmachine/vsphere/vm.go

```diff
@@ -878,15 +878,26 @@ func (vm *VM) Destroy() (err error) {
 		go func() {
 			defer timer.Stop()
 			defer wg.Done()
+
+			powerState := "unInitialized"
```

Comment:
I understand this is to initialize powerstate so that we can use it in error message below. IMO `var powerState string` will be better here.

Reply:
ok i got it now. This is to make the error message clear in case powerstate is not initialized and it is timed out.